test/Concurrency/Runtime/async_task_base_priority.swift

```diff
@@ -0,0 +1,118 @@
+// RUN: %target-run-simple-swift( -Xfrontend -disable-availability-checking %import-libdispatch -parse-as-library) | %FileCheck --dump-input=always %s
```

Comment:
Can this use `StdlibUnittest`, or does that interfere with the priority changes you're looking for?

Reply:
No I think it can, I just didn't know about it.